EventFilter/L1TRawToDigi/plugins/implementations_stage2/BMTFSetup.cc

```diff
@@ -22,39 +22,14 @@ namespace l1t {
         std::array<int, 12> board_out = {{1, 7, 2, 8, 3, 9, 4, 10, 5, 11, 6, 12}};  //these are board_ids per amc_no-1
 
         for (unsigned int i = 1; i <= board_out.size(); i++) {
-          if (i % 2 != 0) {  //maybe this check is not needed
-            res[{i, board_out[i - 1]}] = {PackerFactory::get()->make("stage2::BMTFPackerOutput"),
-                                          PackerFactory::get()->make("stage2::BMTFPackerInputs")};
-          } else {
-            res[{i, board_out[i - 1]}] = {PackerFactory::get()->make("stage2::BMTFPackerOutput"),
-                                          PackerFactory::get()->make("stage2::BMTFPackerInputs")};
+          auto packer_out = std::make_shared<BMTFPackerOutput>();
+          auto packer_in = PackerFactory::get()->make("stage2::BMTFPackerInputs");
+          if (fw >= 2452619552) {  // the 1st Kalman fw-ver value (0x95000160)
```

Comment:
please use constexpr for this magic number.

Reply:
@rekovic have a look at the changes after the last commit
since there is not any calculation to make it done in the compilation time I didn't know what you meant by suggesting to write it as `constexpr`.
So, I wrote this a bit more cpp-wise and human-readable in order to be easily maintainable.
Also, now the fw ver are encoded in hex and are consistent with what we have in the L1-keys.